packages/metro-service/src/asset/android.ts

```diff
@@ -52,3 +53,19 @@ export function getAssetDestPathAndroid(
   const fileName = getResourceIdentifier(asset);
   return path.join(androidFolder, `${fileName}.${asset.type}`);
 }
+
+export function saveAssetsAndroid(
+  assets: ReadonlyArray<AssetData>,
+  _platform: string,
+  _assetsDest: string | undefined,
+  _assetCatalogDest: string | undefined,
+  addAssetToCopy: (
+    asset: AssetData,
+    allowedScales: number[] | undefined,
+    getAssetDestPath: (asset: AssetData, scale: number) => string
+  ) => void
+) {
```

Comment:
Nit: If we make this an arrow function, we can type it properly:
```suggestion
export const saveAssetsAndroid: SaveAssetsPlugin = (
  assets,
  _platform,
  _assetsDest,
  addAssetToCopy
) => {
```